docs/Pipeline-Upload.md

```diff
@@ -0,0 +1,25 @@
+# Custom Pipeline Upload
+
+It's possible to upload custom pipes to DIVE Web through the girder interface.
+
+!!! warning
+    This feature is not yet standardized, and the instructions below may change.
+
+1. Open the girder interface at `/girder` and create a new private folder called `MyPipelines`
+    1. For our demo instance, open [https://viame.kitware.com/girder](https://viame.kitware.com/girder)
+1. Create a new folder in that private folder, and give it a name you'd like to associate with your new pipeline.
+1. Upload one or more files inside your new pipeline subfolder:
+    1. A pipeline file ending in the `.pipe` file extension
+    1. Whatever other model `.zip` files are required by the pipe, named exactly as the appear in your `.pipe` file above.
```

Comment:
I think you want it named as it `they appear inside your` 
typically this is `trained_detector.zip` or something like that for trained detectors.